telco-examples/edge-clusters/cpu-manager/README.md

```diff
@@ -0,0 +1,45 @@
+# Example of Edge Cluster with CPU Manager
+
+## Introduction
+
+This is an example to demonstrate how to deploy an edge cluster with cpu manager enabled in kubernetes for Telco using SUSE ATIP and the fully automated directed network provisioning.
+
+## Configuration
+
+Using the file `telco-examples/edge-clusters/cpu-manager/edge-cluster-cpu-manager.yaml` as an example, you can deploy an edge cluster with cpu manager just modifying the following parameters:
+
+- `${EDGE_CONTROL_PLANE_IP}` -  The IP address to be used as a endpoint for the edge cluster (should match with the kubeapi-server endpoint).
+- `${RESERVED_SYSTEM_CPU}` - The number of CPUs to be reserved for the system to be configured by cpu manager.
```

Comment:
Maybe something more concise, like "The number of CPUs to be reserved for use by cpu manager" (or cpu-manager?)

Reply:
I'd say just modify the second part and keep only `The number of CPUs to be reserved for the system.` I think it's enough to understand it